### DISCUSSION TITLE
Message Smartmirroring

### DISCUSSION THREAD

**Discussion by Lex1982** [BimmerGestalt/AAIdrive#788] [General]
I activated the smartmirroring functionality, I installed the apk on the smartphone.  in the vehicle when I select it, the attached message appears.  How should I authorize smartmirroring.. I use a Samsung S23.  Thank you

![20240523_072919](https://github.com/BimmerGestalt/AAIdrive/assets/113173654/a461fee6-5da2-476a-9c4e-fc459d82a5af)

**Comment by hufman** [comment]
The Screen Mirroring app tries to show a notification to request permission, but new Android versions restrict this permission. 
From the AAIdrive Addons tab, click the Screen Mirroring addon to open its phone interface, which has a button to grant notification access for the future, and a button to grant mirroring access for the current session.
